utility/path.go

```diff
@@ -67,3 +70,35 @@ func BaseFilter(base string, allowed bool) FilterFunc {
 		return (allowed == strings.EqualFold(base, b)), nil
 	}
 }
+
+// FindFileInAppDir ...
+func FindFileInAppDir(appDir, fileName string) (string, error) {
+	filePth := filepath.Join(appDir+".app", fileName)
```

Comment:
appDir should be a path that already contains the .app extension

Reply:
Fixed.